src/test/groovy/smartthings/migration/MigrationRunnerSpec.groovy

```diff
@@ -206,8 +208,8 @@ class MigrationRunnerSpec extends Specification {
 	List<Map> processRows(ResultSet results) {
 		results.all().collect { row ->
 			row.columnDefinitions.collect { column ->
-				assert column.keyspace == keyspace
-				[column.name, row.getString(column.name)]
+				assert column.keyspace.asCql(true) == keyspace
+				[column.name.asCql(true), row.getString(column.name)]
```

Comment:
column.keyspace and column.name no longer strings.  Later cassandra driver introduces a wrapper type CqlIdentifer.

https://github.com/datastax/java-driver/blob/4.x/core/src/main/java/com/datastax/oss/driver/api/core/CqlIdentifier.java#L118